spec/client_spec.rb

```diff
@@ -43,6 +43,17 @@ def loop_times(times)
     @client.update_users(@random_users)
   end
 
+  it 'properly sets up a new client' do
+    client = StreamChat::Client.new(ENV['STREAM_KEY'], ENV['STREAM_SECRET'], base_url: ENV['STREAM_CHAT_URL'])
```

Comment:
Nitpick(not specifically about this MR): this should be really a separate function that will return either configuration from env, or constructed client.

Reply:
✅

Reply:
Thanks man, looks amazing.